packages/genie-ui/client/styles.css

```
@@ -0,0 +1,425 @@
+.chat-from {
+  font-size: 10px;
+  text-transform: uppercase;
+  letter-spacing: 0.04em;
+  color: var(--dim);
+}
+.chat-line.human .chat-from {
+  color: var(--accent);
+}
+.chat-text {
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)